source/Patches/Roles/Role.cs

```diff
@@ -512,6 +512,45 @@ public static class HudManager_Update
             private static Vector3 oldScale = Vector3.zero;
             private static Vector3 oldPosition = Vector3.zero;
 
+            //add the colorType List from DeadBody.cs of MedicMod folder to use here
+            public static String GetColorType(PlayerControl player)
```

Comment:
Is this the only place this is used? Let's move it somewhere closer to the Medic and avoid jamming stuff into utility classes.

Reply:
ok